src/local-time.lisp

```diff
@@ -1387,7 +1387,7 @@ The value of this variable should have the methods `local-time::clock-now', and
     (multiple-value-bind (offset* daylight-p abbreviation)
         (timestamp-subtimezone timestamp timezone)
       (multiple-value-bind (adjusted-secs adjusted-days)
-          (%adjust-to-timezone timestamp timezone offset)
+          (%adjust-to-timezone timestamp timezone (or offset offset*))
```

Comment:
Unclear how this is preventing calls. Later PR?

Reply:
See `%adjust-to-timezone`: it would call timestamp-subtimezone again on the same timestamp and timezone when offset is null.